ruby/app.rb

```diff
@@ -0,0 +1,61 @@
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    puts 'There is no book registered!' if @books.empty?
+    @books.each_with_index do |book, index|
+      puts "#{index}) Title: '#{book.title}', Author: #{book.author}"
+    end
+  end
+
+  def list_people(with_number: false)
+    puts 'There is no people registered!' if @people.empty?
+    @people.each_with_index do |person, index|
+      print "#{index}) " if with_number
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def add_teacher(age, name, specialization)
+    @people << Teacher.new(age, specialization, name)
+  end
+
+  def add_student(age, name, parent_permission)
+    @people << Student.new(age, '', name, parent_permission: parent_permission)
+  end
```

Comment:
- [ ] Good job so far but when I try to create a student the app stop and gives me an error `wrong number of arguments`, please kindly fix that and make sure everything is working as expected 🙃 😉 
- [ ] So as I can't create a person (`teacher and student`) I can't create a rental as well, so please make sure that users can create rentals as well 🙃 🙃 

<img width="1127" alt="Screenshot 2023-04-15 at 20 42 53" src="https://user-images.githubusercontent.com/74501523/232245465-ae7d966e-882c-4fe0-8466-e4a839d88304.png">